shaderc-sys/src/lib.rs

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 #![allow(non_camel_case_types)]
 
-use libc::{c_char, c_int, c_void, size_t, int32_t, uint32_t};
+extern crate libc;
```

Comment:
Just remembered this.  I don't have a clear explanation of why this was needed, just that it broke without the change.